apps/estuary/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     - "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT"
     - "KAFKA_LISTENERS=INSIDE://:9094,OUTSIDE://:9092"
     - "KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE"
-    - "KAFKA_CREATE_TOPICS=clusterready:1:1"
+    - "KAFKA_CREATE_TOPICS=clusterready:1:1,dead-letters:1:1"
```

Comment:
Now that we create the `dead-letters` topic on start of kafka can we remove `clusterready` topic since it's not used anywhere